ui/src/literature/components/BookSeriesInfoList.jsx

```diff
@@ -4,25 +4,22 @@ import { List } from 'immutable';
 
 import InlineList, { SEPARATOR_AND } from '../../common/components/InlineList';
 
-function extractChapter(bookChapter) {
-  return bookChapter.get('title') + bookChapter.get('volume', '');
+function extractSeries(bookSeries) {
+  return bookSeries.get('title') + bookSeries.get('volume', '');
 }
 
-function renderSeriesInfo(bookChapter) {
-  return (
-    <span>
-      {bookChapter.get('title')} volume:{bookChapter.get('volume')}
-    </span>
-  );
+function renderSeriesInfo(bookSeries) {
+  const volume = bookSeries.get('volume');
+  return <span>{bookSeries.get('title') + volume && `, ${volume}`}</span>;
```

Comment:
I don't think this is the right way to do it, suggestions welcome :)

Reply:
sth like that would be better.
```js
<span>{bookSeries.get('title')}{volume && <span>, {volume}</span>}</span>
```